examples/advanced_tensorflow/client.py

```diff
@@ -56,14 +57,16 @@ def evaluate(self, parameters, config):
         # Get config values
         steps: int = config["val_steps"]
 
-        # Evaluate global model parameters on the local test data
+        # Evaluate global model parameters on the local test data and return results
         loss, accuracy = self.model.evaluate(self.x_test, self.y_test, 32, steps=steps)
-
-        # Populate the 3rd return value with a 0.0 float, it'll be removed in the future
-        return len(self.x_test), loss, 0.0, {"accuracy": accuracy}
+        num_examples_test = len(self.x_test)
+        return loss, num_examples_test, {"accuracy": accuracy}
 
 
 def main() -> None:
+    # Sleep for 2s to give the server enough time to start
+    time.sleep(2)
```

Comment:
This will impact each client start in the example. Is that intentional?

Reply:
Yes, that's intentional - I had several issues with the clients trying to connect before the server was ready. Unfortunately the logs do not explain this well, so it can be a mystery for users to understand what's happening. We need to follow up with improved logging at a later point.